### PR TITLE
Enable Caddy HTTP cache

### PR DESCRIPTION
**Pull request by samdark**
One of the advantages of using Caddy/FrankenPHP is its HTTP cache support.

See https://api-platform.com/docs/core/performance/#built-in-caddy-http-cache

### CODE REVIEW

File: images/frankenphp/Caddyfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+    cache
```
Comment (shyim) [inline images/frankenphp/Caddyfile:2]: I would try to keep this file same as https://github.com/dunglas/frankenphp/blob/main/caddy/frankenphp/Caddyfile

You could set environment variable `CADDY_GLOBAL_OPTIONS` to `cache` right?

**Comment by samdark** [comment]
Removed Caddyfile changes.

**Review by shyim — COMMENTED** [review]
increase epoch

**Comment by samdark** [comment]
Thanks for quick review.

**Pull request merged**